lib/jsender/json.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+module Jsender
+  module Json
```

Comment:
Why not keep the changes of your code below( which are much more clearer to read), and retain the prior namespace (Jsender.error etc), so it's non-breaking contract for consumers?

Reply:
I needed a way to differentiate between the two outputs types.  Stuff for Rack and Stuff that is just Json.  Might later want another one for Jsend that we don't want Json encoded... just a ruby hash.  Jsend.error does not tell me what I'll will be getting.  Jsender.Json vs Jsender.Rack tells me what you will be getting.   Also, this is a major version but because these changes are breaking the current Jsend gem contract with customers.   Below you mentioned a module layout that will result in Jsender::Helpers::Json.error and Jsender::Helpers::Rack.error    I'm happy with that.  Should I refactor?  Or is there something you feel we should put in Jsend.error?  Currently (for this pull request changes) there will be no Jsend.error.

Reply:
Makes sense as is. Consider all comments as complete. Let's get some traction with it!

Reply:
:-) Thanks.  Yes, would like to settle this! :-) :-)